experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -1025,7 +1035,7 @@ describe("Editing", () => {
 					parentField: brand("foo"),
 					parentIndex: 0,
 				});
-				tree2.editor.setValue(path, "b");
+				// tree2.editor.setValue(path, "b");
```

Comment:
This edit is important because we check that it doesn't happen (due to the constraint violation). If the constraint violation code stopped working, this test would still pass.
You can replace this with any edit you want so long that we'd notice if it were applied (i.e., it should lead to a state different from `{}`).